README.md

```diff
@@ -96,5 +96,5 @@ Bug reports and pull requests are welcome on GitHub at https://github.com/atomic
 The gem is available as open source under the terms of the [MIT License](http://opensource.org/licenses/MIT).
 
 
-### Things Not Quite Implemented
-People, Groups, Sets, Blogs
+## Things Not Quite Implemented
+People, Sets
```

Comment:
Ok. I'll remove people from the list.

I also wondered if sets was referring to question sets / question banks.